sdk/monitor/monitor-query/samples/v1/javascript/metricsQuery.js

```diff
@@ -21,31 +21,36 @@ async function main() {
     throw new Error("METRICS_RESOURCE_ID must be set in the environment for this sample");
   }
 
-  const result = await metricsQueryClient.getMetricDefinitions(metricsResourceId);
-
-  for (const definition of result.definitions) {
-    console.log(`Definition = ${definition.name}`);
+  const iterator = metricsQueryClient.listMetricDefinitions(metricsResourceId);
+  let result = await iterator.next();
+  const firstMetric = result.value;
+  let secondMetricName;
+  while (!result.done) {
+    console.log(` metricDefinitions - ${result.value.id}, ${result.value.name}`);
+    secondMetricName = result.value.name;
+    result = await iterator.next();
   }
-
-  const firstMetric = result.definitions[0];
+  console.log(`First Metric Definition = ${firstMetric.name}`);
 
   console.log(`Picking an example metric to query: ${firstMetric.name}`);
 
-  const metricsResponse = await metricsQueryClient.queryMetrics(
+  const metricsResponse = await metricsQueryClient.query(
     metricsResourceId,
-    Durations.last5Minutes,
+    [firstMetric.name, secondMetricName],
```

Comment:
`secondMetricName` might be undefined if `listMetricDefinitions` only returns one result. Any reason we can't just collect all the names and pass them to `query`?

Reply:
changed the sample to pass in all metric names